TypeError: response.api_response.body is not a function
    at /usr/src/app/dist/controllers/MercadoPagoController.js:334:126
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:187:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
TypeError: response.api_response.body is not a function
    at /usr/src/app/dist/controllers/MercadoPagoController.js:334:126
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:187:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
TypeError: response.api_response.body is not a function
    at /usr/src/app/dist/controllers/MercadoPagoController.js:334:126
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:187:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/modules/cjs/loader:1137
  throw err;
  ^

Error: Cannot find module 'dotenv'
Require stack:
- /usr/src/app/dist/app.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1134:15)
    at Module._load (node:internal/modules/cjs/loader:975:27)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/app.js:6:34)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/src/app/dist/app.js' ]
}

Node.js v18.19.0
node:internal/modules/cjs/loader:1137
  throw err;
  ^

Error: Cannot find module 'dotenv'
Require stack:
- /usr/src/app/dist/app.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1134:15)
    at Module._load (node:internal/modules/cjs/loader:975:27)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/app.js:6:34)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/src/app/dist/app.js' ]
}

Node.js v18.19.0
node:internal/modules/cjs/loader:1137
  throw err;
  ^

Error: Cannot find module 'dotenv'
Require stack:
- /usr/src/app/dist/app.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1134:15)
    at Module._load (node:internal/modules/cjs/loader:975:27)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/app.js:6:34)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/src/app/dist/app.js' ]
}

Node.js v18.19.0
node:internal/modules/cjs/loader:1137
  throw err;
  ^

Error: Cannot find module 'dotenv'
Require stack:
- /usr/src/app/dist/app.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1134:15)
    at Module._load (node:internal/modules/cjs/loader:975:27)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/app.js:6:34)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/src/app/dist/app.js' ]
}

Node.js v18.19.0
TypeError: response.api_response.body is not a function
    at /usr/src/app/dist/controllers/MercadoPagoController.js:334:126
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:187:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
TypeError: response.api_response.body is not a function
    at /usr/src/app/dist/controllers/MercadoPagoController.js:334:126
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:187:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:494
ah;
^

ReferenceError: ah is not defined
    at Object.<anonymous> (/usr/src/app/dist/controllers/MercadoPagoController.js:494:1)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:186:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:186:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:188:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:188:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:187:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:187:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:186:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:186:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
/usr/src/app/dist/routes/paymentsRoute.js:65
back_urls;
^

ReferenceError: back_urls is not defined
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:65:1)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:19:41)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)

Node.js v18.19.0
/usr/src/app/dist/routes/paymentsRoute.js:65
back_urls;
^

ReferenceError: back_urls is not defined
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:65:1)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:19:41)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)

Node.js v18.19.0
/usr/src/app/dist/routes/paymentsRoute.js:67
router.get('/mercadopago/back_url', [], backUrl);
                                        ^

ReferenceError: backUrl is not defined
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:67:41)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:19:41)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)

Node.js v18.19.0
/usr/src/app/dist/routes/paymentsRoute.js:67
router.get('/mercadopago/back_url', [], backUrl);
                                        ^

ReferenceError: backUrl is not defined
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:67:41)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:19:41)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:575
('/mercadopago/notification_url', [], notificationUrl);
                                      ^

ReferenceError: notificationUrl is not defined
    at Object.<anonymous> (/usr/src/app/dist/controllers/MercadoPagoController.js:575:39)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:575
('/mercadopago/notification_url', [], notificationUrl);
                                      ^

ReferenceError: notificationUrl is not defined
    at Object.<anonymous> (/usr/src/app/dist/controllers/MercadoPagoController.js:575:39)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)

Node.js v18.19.0
{
  message: 'auto_return invalid. back_url.success must be defined',
  error: 'invalid_auto_return',
  status: 400,
  cause: null
}
{
  message: 'auto_return invalid. back_url.success must be defined',
  error: 'invalid_auto_return',
  status: 400,
  cause: null
}
{
  message: 'auto_return invalid. back_url.success must be defined',
  error: 'invalid_auto_return',
  status: 400,
  cause: null
}
{
  message: 'auto_return invalid. back_url.success must be defined',
  error: 'invalid_auto_return',
  status: 400,
  cause: null
}
{
  message: 'auto_return invalid. back_url.success must be defined',
  error: 'invalid_auto_return',
  status: 400,
  cause: null
}
{
  message: 'auto_return invalid. back_url.success must be defined',
  error: 'invalid_auto_return',
  status: 400,
  cause: null
}
/usr/src/app/dist/controllers/MercadoPagoController.js:575
        console.log({ 'backUrl': , params: req === null || req === void 0 ? void 0 : req.params, query: req === null || req === void 0 ? void 0 : req.query, body: req === null || req === void 0 ? void 0 : req.body, path: req === null || req === void 0 ? void 0 : req.path });
                                 ^

SyntaxError: Unexpected token ','
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:575
        console.log({ 'backUrl': , params: req === null || req === void 0 ? void 0 : req.params, query: req === null || req === void 0 ? void 0 : req.query, body: req === null || req === void 0 ? void 0 : req.body, path: req === null || req === void 0 ? void 0 : req.path });
                                 ^

SyntaxError: Unexpected token ','
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:26
    accessToken: privateKey,
                 ^

ReferenceError: privateKey is not defined
    at Object.<anonymous> (/usr/src/app/dist/controllers/MercadoPagoController.js:26:18)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:594
const notificationUrl = (req, res) => __awaiter(void 0, void 0, void 0, function* () {
      ^

SyntaxError: Identifier 'notificationUrl' has already been declared
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:597
        const paymentData = yield retrievePayment(query. ? payment_id : );
                                                         ^

SyntaxError: Unexpected token '?'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:597
        const paymentData = yield retrievePayment(query. ? payment_id : );
                                                         ^

SyntaxError: Unexpected token '?'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
undefined:1
ReferenceError: postData is not defined
^

SyntaxError: Unexpected token R in JSON at position 0
    at JSON.parse (<anonymous>)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:295:27
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.19.0
undefined:1
ReferenceError: postData is not defined
^

SyntaxError: Unexpected token R in JSON at position 0
    at JSON.parse (<anonymous>)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:295:27
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.19.0
undefined:1
ReferenceError: postData is not defined
^

SyntaxError: Unexpected token R in JSON at position 0
    at JSON.parse (<anonymous>)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:295:27
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.19.0
undefined:1
ReferenceError: postData is not defined
^

SyntaxError: Unexpected token R in JSON at position 0
    at JSON.parse (<anonymous>)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:295:27
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.19.0
undefined:1
ReferenceError: postData is not defined
^

SyntaxError: Unexpected token R in JSON at position 0
    at JSON.parse (<anonymous>)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:295:27
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.19.0
undefined:1
ReferenceError: postData is not defined
^

SyntaxError: Unexpected token R in JSON at position 0
    at JSON.parse (<anonymous>)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:295:27
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:647
    catch () { }
           ^

SyntaxError: Unexpected token ')'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:647
    catch () { }
           ^

SyntaxError: Unexpected token ')'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:647
    catch () { }
           ^

SyntaxError: Unexpected token ')'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:647
    catch () { }
           ^

SyntaxError: Unexpected token ')'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:225
        * /;
        ^

SyntaxError: Unexpected token '*'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/PaymentIntentController.js:148
        res.status(500).json({
            ^

TypeError: res.status is not a function
    at /usr/src/app/dist/controllers/PaymentIntentController.js:148:13
    at Generator.next (<anonymous>)
    at /usr/src/app/dist/controllers/PaymentIntentController.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/src/app/dist/controllers/PaymentIntentController.js:4:12)
    at store (/usr/src/app/dist/controllers/PaymentIntentController.js:40:29)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:601:45
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.19.0
/usr/src/app/dist/controllers/PaymentIntentController.js:148
        res.status(500).json({
            ^

TypeError: res.status is not a function
    at /usr/src/app/dist/controllers/PaymentIntentController.js:148:13
    at Generator.next (<anonymous>)
    at /usr/src/app/dist/controllers/PaymentIntentController.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/src/app/dist/controllers/PaymentIntentController.js:4:12)
    at store (/usr/src/app/dist/controllers/PaymentIntentController.js:40:29)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:601:45
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.19.0
/usr/src/app/dist/controllers/PaymentIntentController.js:148
        res.status(500).json({
            ^

TypeError: res.status is not a function
    at /usr/src/app/dist/controllers/PaymentIntentController.js:148:13
    at Generator.next (<anonymous>)
    at /usr/src/app/dist/controllers/PaymentIntentController.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/src/app/dist/controllers/PaymentIntentController.js:4:12)
    at store (/usr/src/app/dist/controllers/PaymentIntentController.js:40:29)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:601:45
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.19.0
/usr/src/app/dist/controllers/PaymentIntentController.js:148
        res.status(500).json({
            ^

TypeError: res.status is not a function
    at /usr/src/app/dist/controllers/PaymentIntentController.js:148:13
    at Generator.next (<anonymous>)
    at /usr/src/app/dist/controllers/PaymentIntentController.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/src/app/dist/controllers/PaymentIntentController.js:4:12)
    at store (/usr/src/app/dist/controllers/PaymentIntentController.js:40:29)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:601:45
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.19.0
/usr/src/app/dist/controllers/PaymentIntentController.js:148
        res.status(500).json({
            ^

TypeError: Cannot read properties of null (reading 'status')
    at /usr/src/app/dist/controllers/PaymentIntentController.js:148:13
    at Generator.next (<anonymous>)
    at /usr/src/app/dist/controllers/PaymentIntentController.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/src/app/dist/controllers/PaymentIntentController.js:4:12)
    at store (/usr/src/app/dist/controllers/PaymentIntentController.js:40:29)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:601:45
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:601
        const paymentData = await retrievePayment(query === null || query === void 0 ? void 0 : query.payment_id);
                            ^^^^^

SyntaxError: await is only valid in async functions and the top level bodies of modules
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:600
        const paymentData = await retrievePayment(query === null || query === void 0 ? void 0 : query.payment_id);
                            ^^^^^

SyntaxError: await is only valid in async functions and the top level bodies of modules
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:599
        const paymentData = yield retrievePayment(req. ? query === null || query === void 0 ? void 0 : query.payment_id : );
                                                       ^

SyntaxError: Unexpected token '?'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:600
        const paymentData = yield retrievePayment(req. ? query === null || query === void 0 ? void 0 : query.payment_id : );
                                                       ^

SyntaxError: Unexpected token '?'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:600
        const paymentData = yield retrievePayment(req. ? query === null || query === void 0 ? void 0 : query.payment_id : );
                                                       ^

SyntaxError: Unexpected token '?'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:600
        const paymentData = yield retrievePayment(req. ? query === null || query === void 0 ? void 0 : query.payment_id : );
                                                       ^

SyntaxError: Unexpected token '?'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:600
        const paymentData = yield retrievePayment(req. ? query === null || query === void 0 ? void 0 : query.payment_id : );
                                                       ^

SyntaxError: Unexpected token '?'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:600
        const paymentData = yield retrievePayment(req. ? query === null || query === void 0 ? void 0 : query.payment_id : );
                                                       ^

SyntaxError: Unexpected token '?'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:600
        const paymentData = yield retrievePayment(req. ? query === null || query === void 0 ? void 0 : query.payment_id : );
                                                       ^

SyntaxError: Unexpected token '?'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:600
        const paymentData = yield retrievePayment(req. ? query === null || query === void 0 ? void 0 : query.payment_id : );
                                                       ^

SyntaxError: Unexpected token '?'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/MercadoPagoController.js:600
        const paymentData = yield retrievePayment(req. ? query === null || query === void 0 ? void 0 : query.payment_id : );
                                                       ^

SyntaxError: Unexpected token '?'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/routes/paymentsRoute.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:174:12)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:278:15)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:405:5)
    at ServerResponse.setHeader (node:_http_outgoing:648:11)
    at ServerResponse.header (/usr/src/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:275:10)
    at /usr/src/app/dist/controllers/MercadoPagoController.js:582:25
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/controllers/MercadoPagoController.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v18.19.0
Wed, 17 Jan 2024 15:53:26 GMT express deprecated res.send(status): Use res.sendStatus(status) instead at dist/controllers/MercadoPagoController.js:330:13
/usr/src/app/dist/controllers/PaymentIntentController.js:23
    res.status(200)..json({ data });
                    ^

SyntaxError: Unexpected token '.'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/controllers/StripeProductController.js:43:35)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/dist/controllers/PaymentIntentController.js:234
    console.log(error);
                ^

ReferenceError: error is not defined
    at Object.<anonymous> (/usr/src/app/dist/controllers/PaymentIntentController.js:234:17)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/controllers/StripeProductController.js:39:35)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)

Node.js v18.19.0
/usr/src/app/dist/controllers/PaymentIntentController.js:234
    console.log(error);
                ^

ReferenceError: error is not defined
    at Object.<anonymous> (/usr/src/app/dist/controllers/PaymentIntentController.js:234:17)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/controllers/StripeProductController.js:39:35)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)

Node.js v18.19.0
/usr/src/app/dist/controllers/PaymentIntentController.js:230
    catch () { }
           ^

SyntaxError: Unexpected token ')'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/controllers/StripeProductController.js:39:35)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/PaymentIntentController.js:230
    catch () { }
           ^

SyntaxError: Unexpected token ')'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/controllers/StripeProductController.js:39:35)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/PaymentIntentController.js:230
    catch () { }
           ^

SyntaxError: Unexpected token ')'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/controllers/StripeProductController.js:39:35)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/PaymentIntentController.js:234
    console.log(error);
                ^

ReferenceError: error is not defined
    at Object.<anonymous> (/usr/src/app/dist/controllers/PaymentIntentController.js:234:17)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/controllers/StripeProductController.js:39:35)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
(node:7) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 exit listeners added to [Bus]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/dist/controllers/PaymentIntentController.js:18
imposr;
^

ReferenceError: imposr is not defined
    at Object.<anonymous> (/usr/src/app/dist/controllers/PaymentIntentController.js:18:1)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/controllers/StripeProductController.js:39:35)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)

Node.js v18.19.0
node:internal/validators:162
    throw new ERR_INVALID_ARG_TYPE(name, 'string', value);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "id" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:405:5)
    at validateString (node:internal/validators:162:11)
    at Module.require (node:internal/modules/cjs/loader:1218:3)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/controllers/PaymentIntentController.js:18:34)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19) {
  code: 'ERR_INVALID_ARG_TYPE'
}

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/dist/controllers/PaymentIntentController.js:234
            const info = yield transport.sendMail({
                         ^^^^^

SyntaxError: Unexpected strict mode reserved word
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/controllers/StripeProductController.js:39:35)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/dist/controllers/PaymentIntentController.js:233
            const info = yield transport.sendMail({
                         ^^^^^

SyntaxError: Unexpected strict mode reserved word
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1274:20)
    at Module._compile (node:internal/modules/cjs/loader:1320:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/controllers/StripeProductController.js:39:35)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
(node:7) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 exit listeners added to [Bus]. Use emitter.setMaxListeners() to increase limit
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as post] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:10:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as get] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:13:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.put() requires a callback function but got a [object Undefined]
    at Route.<computed> [as put] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as put] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:16:8)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
/usr/src/app/node_modules/express/lib/router/route.js:211
        throw new Error(msg);
        ^

Error: Route.delete() requires a callback function but got a [object Undefined]
    at Route.<computed> [as delete] (/usr/src/app/node_modules/express/lib/router/route.js:211:15)
    at proto.<computed> [as delete] (/usr/src/app/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/usr/src/app/dist/routes/StripeProductRoute.js:21:14)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/usr/src/app/dist/models/server.js:26:46)

Node.js v18.19.0
